auth-service/src/main/java/com/petclinic/auth/User/UserController.java

```diff
@@ -62,4 +62,11 @@ public void passwordReset(@PathVariable long id,  @RequestBody String pwd) throw
         userServ.passwordReset(id,pwd);
         log.info("Password for User with id {} with new password {}", id, pwd);
     }
+
+    @GetMapping
+     public Boolean verifyUser (@RequestBody UserIDLessUsernameLessDTO loginUser) throws NotFoundException{
```

Comment:
if you want to verify if the user is registered, why request the body when you can request the ID and verify it inside the database?

Reply:
Unless you verify by other values such as the username.